completion/functions/compstyle_zsh-utils_setup

```diff
@@ -0,0 +1,93 @@
+#!/bin/zsh
+
+function compstyle_zsh-utils_help () {
+  cat <<'EOF'
+This completion style is from the zsh-utils project.
+You can invoke it with the following command:
+
+  compstyle zsh-utils
+
+More information available here: https://github.com/belak/zsh-utils
+EOF
+}
+
+function compstyle_zsh-utils_setup {
+  # Defaults.
+  zstyle ':completion:*:default' list-colors ${(s.:.)LS_COLORS}
+  zstyle ':completion:*:default' list-prompt '%S%M matches%s'
+
+  # Use caching to make completion for commands such as dpkg and apt usable.
+  zstyle ':completion::complete:*' use-cache on
+  zstyle ':completion::complete:*' cache-path "${XDG_CACHE_HOME:-$HOME/.cache}/zsh/zcompcache"
```

Comment:
I feel like the cache-related settings are fairly universally applicable, so we can leave them in the completion plugin and remove them from here.

EDIT: I just noticed that the compstyle cleanup removes all completion zstyles... so maybe there's a better way to do this? I'm open to suggestions. Maybe completion should also export the compstyle location as an environment variable to make sure it matches.

Reply:
Oh yeah - forgot to mention that. When I added the cleanup functions I had to have a way to re-apply the cache zstyles, so I just added them to the setup file. I don't have to be so aggressive with the cleanup, but I was going for simplicity. Or, we could punt on cleanup functionality for now as originally discussed.

Reply:
Maybe we could export a variable for the cache location and use that in `compstyle` to re-apply these 2 settings after cleanup?

Reply:
Resolved in recent commit.